Backend/chat/chatController.js

```diff
@@ -0,0 +1,41 @@
+const chatModel = require("./chatModel.js");
+const io = require("../socket");
+
+exports.getAllMessages = async (req, res) => {
+  const { order_id } = req.params;
+  try {
+    const allMyOrders = await chatModel.findAllByOrderId(order_id);
+    if (!allMyOrders) {
+      return res.status(400).json({
+        errorMessage: "Something went wrong with your chat request",
+      });
+    }
+    return res.status(200).json(allMyOrders);
+  } catch (error) {
```

Comment:
check

Reply:
We can’t make separate one block(express-error-handler) for try-catch this is because there are a lot of different statements of each try-catch block. If there are few statements of each try block then we can pass by props but there are a lot of different types of statements each having different types of conditions and different types of error messages so it will be a mess if we send all different types of statements via long props.
 
Also, each try block has different type of blocking and non-blocking functions so there will be a lot of conditions which cause of increase the lines of code instead of a decrease. All big companies follow this pattern of each block has itself logic, statements and conditions so we always keep it own try-catch block yo. Does it make sense?

Reply:
Check